docs/download.md

```diff
@@ -18,6 +18,62 @@ cuDF jar, that is either preinstalled in the Spark classpath on all nodes or sub
 that uses the RAPIDS Accelerator For Apache Spark. See the [getting-started
 guide](https://nvidia.github.io/spark-rapids/Getting-Started/) for more details.
 
+## Release v21.06.0
+Starting with release 21.06.0, the project is moving to calendar versioning, with the first two
+digits representing the year, the second two digits representing the month, and the last digit
+representing the patch version of the release. 
+
+Hardware Requirements: 
+
+	GPU Architecture: NVIDIA V100, T4 and A30/A100 GPUs
+
+Software Requirements:
+
+	OS: Ubuntu 18.04, Ubuntu 20.04 or CentOS 7, CentOS8
```

Comment:
Do we support Ubuntu 16.04 as cuDF does?

Reply:
We do not test on Ubuntu 16.04, so we do not mention it in the supported list.